src/cli/commands/help.js

```diff
@@ -41,11 +43,14 @@ export function run(config: Config, reporter: Reporter, commander: Object, args:
     const getDocsLink = name => `${constants.YARN_DOCS}${name || ''}`;
     console.log('  Commands:\n');
     for (const name of Object.keys(commands).sort(sortAlpha)) {
-      if (commands[name].useless) {
+      if (commands[name].useless || unsupportedAliases[name] || Object.values(aliases).includes(name)) {
         continue;
       }
-
-      console.log(`    - ${hyphenate(name)}`);
+      if (aliases[name]) {
+        console.log(`    - ${hyphenate(name)}  alias: ${aliases[name]}`);
```

Comment:
I agree, that will get the job done and would look cleaner as well. I will update the PR accordingly.